.github/workflows/tests.yaml

```diff
@@ -4,67 +4,63 @@ on:
   push:
     branches: [main]
   pull_request:
-    branches: [main]
+    branches:
+      - "*"
   schedule:
     - cron: "0 0 * * 1,4"
 
 jobs:
-  Linting:
```

Comment:
So we will no longer be doing a formatting/linting check in CI?

Reply:
I have enabled pre-commit.ci instead. See the checks below.